tokio-quiche/src/http3/driver/server.rs

```diff
@@ -130,6 +160,12 @@ impl ServerHooks {
             stream_ctx.associated_dgram_flow_id = Some(flow_id);
         }
 
+        let latest_priority_update: Option<RawPriorityValue> = driver
+            .conn_mut()?
+            .take_last_priority_update(stream_id)
+            .ok()
+            .map(|v| v.into());
```

Comment:
I think this is the main functional change in this PR.  But I think there are no assertions that would detect if you were to change the right hand side of this expression to `None`

I assume there are planned followup changes to use this priority information.

Reply:
Yeah, the difficulty is in needing a client that could send the PRIORITY_UPDATE frame. That's not something we expose in the TQ client already and I something that is tangential to current needs. It is possible in h3i but adding that as a test capability seems a lot of work for the benefit t might provide. Not sure if @evanrittenhouse has considered h3i here before.